common/src/main/java/com/sequenceiq/cloudbreak/quartz/statuschecker/service/StatusCheckerJobService.java

```diff
@@ -93,22 +108,26 @@ public void unschedule(String id) {
     public void deleteAll() {
         try {
             scheduler.clear();
+            LOGGER.info("All scheduled tasks are cleared");
         } catch (SchedulerException e) {
             LOGGER.error("Error during clearing quartz jobs", e);
         }
     }
 
-    private void schedule(JobDetail jobDetail, Trigger trigger, String localId) {
+    private void schedule(JobDetail jobDetail, Trigger trigger, JobResource jobResource) {
         try {
-            JobKey jobKey = JobKey.jobKey(localId, JOB_GROUP);
+            JobKey jobKey = JobKey.jobKey(jobResource.getLocalId(), JOB_GROUP);
             if (scheduler.getJobDetail(jobKey) != null) {
                 LOGGER.info("Unscheduling status checker job for stack with key: '{}' and group: '{}'", jobKey.getName(), jobKey.getGroup());
-                unschedule(localId);
+                unschedule(jobResource.getLocalId());
             }
             LOGGER.info("Scheduling status checker job for stack with key: '{}' and group: '{}'", jobKey.getName(), jobKey.getGroup());
             scheduler.scheduleJob(jobDetail, trigger);
+            LOGGER.info("Status checker job scheduled with id: {}, name: {}, crn: {}, type: {}", jobResource.getLocalId(), jobResource.getName(),
+                    jobResource.getRemoteResourceId(), jobDetail.getJobDataMap().get(SYNC_JOB_TYPE));
         } catch (SchedulerException e) {
-            LOGGER.error(String.format("Error during scheduling quartz job: %s", localId), e);
+            LOGGER.error("Error during scheduling quartz job. id: {}, name: {}, crn: {}", jobResource.getLocalId(), jobResource.getName(),
```

Comment:
It would be more practical to implement the toString() of the jobResource, and use that in logs, like:
`LOGGER.error("Error during scheduling quartz job with resource {}., jobResource)`

Reply:
JobResource is an interface, that cannot implement the `toString()`